docs/framework/deployment/client-profile.md

```diff
@@ -40,4 +40,4 @@ The .NET Client Profile is a subset of the .NET Framework, which was provided wi
  [.NET Framework Client Profile (.NET Framework 4)](http://msdn.microsoft.com/library/cc656912\(v=vs.100\).aspx)   
  [Visual Studio Multi-Targeting Overview](http://msdn.microsoft.com/library/b1702c33-0672-4ebc-b779-2b324d6ea880)   
  [Troubleshooting .NET Framework Targeting Errors](http://msdn.microsoft.com/library/830e3e45-9a93-4279-a249-75b84599aefb)   
- [How to: Target a Version of the .NET Framework](http://msdn.microsoft.com/library/dea62d25-3d1b-492e-a6cc-b5154489800a)
\ No newline at end of file
+ [How to: Target a Version of the .NET Framework](/visualstudio/ide/how-to-target-a-version-of-the-dotnet-framework)
```

Comment:
This is just an artifact that the branch doesn't contain the VS content. If you add branch=master for example, it works. So it should work when published.